src/alephbet.coffee

```diff
@@ -24,21 +25,14 @@ class AlephBet
     @goal_complete: (experiment_name, variant, goal) =>
       @_track(@namespace, "#{experiment_name} | #{variant}", goal)
 
-  class @LocalStorageAdapter
-    @namespace: 'alephbet'
-    @set: (key, value) ->
-      new Storage(@namespace).set(key, value)
-    @get: (key) ->
-      new Storage(@namespace).get(key)
-
   class @Experiment
     @_options:
       name: null
       variants: null
       sample: 1.0
       trigger: -> true
       tracking_adapter: AlephBet.GoogleUniversalAnalyticsAdapter
```

Comment:
see comment above. For consistency this should be moved, but we can do this separately.